test/integration/scheduler/main_test.go

```diff
@@ -17,11 +17,190 @@ limitations under the License.
 package scheduler
 
 import (
+	"crypto/tls"
+	"crypto/x509"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"path"
+	"strings"
 	"testing"
 
+	"k8s.io/klog/v2/ktesting"
+	kubeapiservertesting "k8s.io/kubernetes/cmd/kube-apiserver/app/testing"
+	kubeschedulertesting "k8s.io/kubernetes/cmd/kube-scheduler/app/testing"
 	"k8s.io/kubernetes/test/integration/framework"
 )
 
 func TestMain(m *testing.M) {
 	framework.EtcdMain(m.Run)
 }
+
+func TestHealthEndpoints(t *testing.T) {
```

Comment:
Can we move to a new folder **serving** under /test/integration/scheduler

Reply:
Done.